packages/portal/src/app/shared/favorite.service.ts

```diff
@@ -13,7 +16,7 @@ export class FavoriteService {
         user,
       }),
     });
-
+    this.realtimeService.broadcastFavouriteNotification(listing);
     return resource.status === 200;
```

Comment:
For extensibility purposes... can that broadcast method be passed items that could be of different types? The we can just call it broadcast